packages/shorebird_code_push_protocol/lib/src/models/create_patch_metadata.dart

```diff
@@ -24,6 +24,7 @@ class CreatePatchMetadata extends Equatable {
     required this.usedIgnoreNativeChangesFlag,
     required this.hasNativeChanges,
     required this.environment,
+    this.linkPercentage,
```

Comment:
The other parameters are required even when not always applicable to ensure we don't forget to provide them when they are applicable. Not blocking but fyi